src/AsyncWebSocket.cpp

```diff
@@ -725,6 +725,22 @@ void AsyncWebSocket::text(uint32_t id, uint8_t * message, size_t len){
 void AsyncWebSocket::text(uint32_t id, char * message){
   text(id, message, strlen(message));
 }
+void AsyncWebSocket::text(uint32_t id, const __FlashStringHelper *data) {
```

Comment:
those method should be added to the AsyncWebSocketClient and pointed to like the other text and binary methods.